graphicsmagick-sys/build.rs

```diff
@@ -35,19 +36,24 @@ fn new_graphicsmagick_config() -> anyhow::Result<GraphicsMagickConfig> {
     };
 
     let content = String::from_utf8(output.stdout)?;
-    for line in content.lines() {
-        if line.starts_with("-I") {
-            gmc.include_flags.push(line.trim().to_string());
-        } else if line.starts_with("-L") {
-            gmc.searches.extend(
-                line.trim()
-                    .split(' ')
-                    .filter(|item| item.starts_with("-L"))
-                    .map(|item| String::from(&item[2..])),
-            )
-        } else if line.starts_with("-l") {
-            gmc.libs
-                .extend(line.trim().split(' ').map(|item| String::from(&item[2..])));
+
+    // split_ascii_whitespace would split '\n', '\t' and whitespace.
+    // It will also filter out any empty str in the result.
+    let mut it = content.split_ascii_whitespace();
+    while let Some(token) = it.next() {
+        let (flag, value) = if ["-I", "-L", "-l"].contains(&token) {
+            // Since token only contains the flag, the next element of it
+            // must contains the value.
+            (token, it.next().unwrap())
+        } else {
+            (&token[..2], &token[2..])
+        };
+
+        match flag {
+            "-I" => gmc.include_flags.push(format!("-I{value}")),
```

Comment:
The `format!("-I{value}"` is the new version rust feature, please update the `Cargo.toml` `rust-version` and README label.

Reply:
I've replaced this with `format!("-I{}", value)`

Reply:
:+1: